ui_framework/components/card/_card_group.scss

```diff
@@ -1,23 +1,18 @@
 /**
- * 1. Use this container to offset the spacing between wrapped cards.
- */
-.kuiCardGroupContainer {
-  margin: -$cardSpacing; /* 1 */
-}
-
-/**
  * 1. Wrap cards when necessary.
+ * 2. Offset the spacing between wrapped cards.
  */
 .kuiCardGroup {
   display: flex;
   flex-wrap: wrap; /* 1 */
+  margin: -$cardSpacing; /* 2 */
 
   /**
    * 1. Use the defined width of the card to determine when to wrap.
    * 2. Use an even margin all around the card so that the spacing is still even when wrapped.
    */
   .kuiCard {
```

Comment:
BEM is good and known, that's why I can read your SCSS
and diverging from a standard needs explanation.
The css_style_guide.md looks like the right place for that.